luminance-gl/src/gl33/pipeline.rs

```diff
@@ -127,6 +130,8 @@ where
       gl::Clear(color_bit | depth_bit);
     }
 
+    gl::Enable(gl::SCISSOR_TEST);
```

Comment:
2. and this?

Reply:
When clearing, the scissor region is taken into account and before was not clearing the whole screen region, leading to parts of the screen being uninitialized and never cleared. ~~Is there a better way to go about it?~~

I suppose the scissor state shouldn't be persisting after the tess gate and needs to be invalidated somehow? Investigating now.